package.json

```diff
@@ -6,8 +6,16 @@
   "preferGlobal": false,
   "main": "./app/server/app.js",
   "scripts": {
-    "clean": "rm -rf node_modules/ dist/ coverage/ npm-debug.log .git.json",
-    "dev": "BUNYAN_LOGLEVEL=debug npm start | bunyan",
+    "clean": "rm -rf node_modules/ dist/ coverage/ npm-debug.log .git.json DockerFile .dockerignore docker-compose.yml",
+    "dev": "BUNYAN_LOG_LEVEL=debug EXPRESS_LOG_FORMAT=dev npm start | bunyan",
+    "docker": "npm run docker-prep && npm run docker-rebuild && npm run docker-up",
+    "docker-build": "docker build -t utopias/hello-world:0.0.1 -f ./Dockerfile .",
```

Comment:
This should be written as a task, (ie/ docker:build) instead of dash notation (docker-build)

Ref: http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/